README.rst

```diff
@@ -34,24 +34,34 @@ Usage
 
 For more in depth info please check usage with ``codespell -h``.
 
-Some noteworthy flags::
+Some noteworthy flags:
+
+::
```

Comment:
Just a thought, but for code highlighting, could specify the parser:

```suggestion
.. code-block:: sh
```

Reply:
Great idea! I'll do that.

Reply:
Done with c0f049aa.